lib/omniauth/strategies/facebook.rb

```diff
@@ -9,18 +9,15 @@ module Strategies
     class Facebook < OmniAuth::Strategies::OAuth2
       class NoAuthorizationCodeError < StandardError; end
 
-      DEFAULT_SCOPE = 'email'
+      DEFAULT_SCOPE = 'email'.freeze
+      API_VERSION = 'v2.6'.freeze
```

Comment:
Fair enough, but this change is orthogonal to versioning the API. Goal is to make this diff to minimal. Not a big deal, thanks for taking the effort regardless.